mne/time_frequency/tests/test_tfr.py

```diff
@@ -639,5 +640,46 @@ def test_compute_tfr():
                                output='avg_power', n_cycles=2.)
             assert_array_equal(shape[1:], out.shape)
 
-
+def test_getitem_epochsTFR():
+    from pandas import DataFrame
+    # Setup for reading the raw data and select a few trials
+    raw = read_raw_fif(raw_fname)
+    events = read_events(event_fname)
+    n_events = 10
+
+    # create fake metadata
+    rt = np.random.uniform(size=(n_events,))
+    trialtypes = np.array(['face', 'place'])
+    trial = trialtypes[(np.random.uniform(size=(n_events,)) > .5).astype(int)]
+    meta = DataFrame(dict(RT=rt, Trial=trial))
+    event_id = dict(a=1, b=2, c=3, d=4)
+    epochs = Epochs(raw, events[:n_events], event_id=event_id, metadata=meta,
+                    decim=2)
+
+    freqs = np.arange(12., 17., 2.)  # define frequencies of interest
+    n_cycles = freqs / 2.  # 0.5 second time windows for all frequencies
+
+    # Choose time x (full) bandwidth product
+    time_bandwidth = 4.0  # With 0.5 s time windows, this gives 8 Hz smoothing
+    power = tfr_multitaper(epochs, freqs=freqs, n_cycles=n_cycles,
+                          use_fft=True, time_bandwidth=time_bandwidth,
+                          return_itc=False, average=False, n_jobs=1)
```

Comment:
you should test all the methods from the mixin class such as __len__ and the iteration with __iter__ and next